### PR TITLE
Doc pytest plugin

### PR DESCRIPTION
**Pull request by graingert**
*Note*: Before submitting this pull request, please review our [contributing
guidelines](https://docs.celeryproject.org/en/master/contributing.html).

### CODE REVIEW

File: docs/userguide/testing.rst [hunk 1]
```diff
@@ -88,14 +88,23 @@ in this example:
             with raises(Retry):
                 send_order(product.pk, 3, Decimal(30.6))
 
-Py.test
+pytest
 =======
```
Comment (graingert) [inline docs/userguide/testing.rst:92]:
```suggestion
======
```

**Comment by graingert** [comment]
@auvipy https://github.com/graingert/pytest-celery/invitations

**Comment by auvipy** [comment]
I dont have access to pc right now. Will accept the invitation soon.

**Comment by thedrow** [comment]
You should invite me too.

**Review by thedrow — APPROVED** [review]
LGTM.

**Comment by auvipy** [comment]
plz invite Omer as well

**Review by auvipy — APPROVED** [review]

**Comment by auvipy** [comment]
thanks for the great effort.

**Comment by graingert** [comment]
related: https://github.com/tmp-org-to-move-pytest-celery-to-celery/pytest-celery/issues/1

**Pull request merged**